### PR TITLE
Rebalances & reorganizes cargo crates

### PR DESCRIPTION
**Pull request by tmtmtl30**
## About The Pull Request

The cargo catalog is a holdover from station-focused design, with unnecessary packs, cluttered categories, and bizarre prices. This PR aims to fix some of these issues, by recategorizing and repricing most packs. Access restrictions on packs have been removed. A few changes have been made to selling cargo as well, with material export prices decreased across the board (quartered, typically) along with the removal of crate and manifest exports. Manifest errors have also been removed. A unit test has been added to catch packs that sell for more than they are bought for.

## Why It's Good For The Game

Credits should be more important, but despite a glut of cargo packs to buy, they often go unspent. A better cargo catalogue is the first step in this direction, even though it may not be perfectly priced. 

## Changelog

:cl:
tweak: The outpost cargo catalog has been recategorized entirely, and should hopefully be more user-friendly.
tweak: Access restrictions have been removed from purchased crates.
tweak: Purchased crates may no longer have erroneous manifests; they will now always contain the full contents of the purchased pack.
tweak: Internals and plasmaman refill crates no longer come with air; instead, purchase a canister of gas along with your tanks.
balance: Many packs have had their prices changed to better fit our ideal price-scaling.
balance: Material export prices have been reduced across the board, with plasma sheets in particular receiving a heavy nerf to their export price.
refactor: Cargo pack categories now exist in their own files.
/:cl:

### CODE REVIEW

**Comment by tmtmtl30** [comment]
death

**Comment by martinlyra** [comment]
Any% merge-conflict speedrun

File: code/modules/unit_tests/supply_pack.dm [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+		var/datum/supply_pack/pack = new pack_type
+		var/created = pack.generate(run_loc_bottom_left)
+
+		var/datum/export_report/rep = export_item_and_contents(created, allowed_categories = ALL, apply_elastic = FALSE)
+		var/value = 0
+		for(var/thing in rep.total_value)
+			value += rep.total_value[thing]
+
+		if(value >= pack.cost)
+			Fail("[pack] ([pack_type]) was resold for [value], [value - pack.cost] greater than the buy price of [pack.cost]!")
```
Comment (ZephyrTFA) [inline code/modules/unit_tests/supply_pack.dm:12]: This also causes a Fail when the resell value is the same as the purchase cost. But the fail reason will report that as '0 greater than the buy price'

**Comment by Zevotech** [comment]
why did you reduce the radioactive protection of biosuits and the fire protection of bomb suits?

**Comment by tmtmtl30** [comment]
their unusually high values decreased the relative usefulness of the other protective suits, making them less worth buying. eventually i want more room for environmental hazards and having specialized protection is a step in that direction

**Comment by Zevotech** [comment]
EXOCON crates are gone, why?

**Comment by Sun-Soaked** [comment]
:c

**Comment by tmtmtl30** [comment]
hate their balance, and really the balance of most mining equipment in general. the hardsuit is too good and the medipens are too good and crushers are too good. we need to move away from /tg/ Gamer Mining

**Comment by Zevotech** [comment]
Great, but you didnt add ANY replacement for them in the catalog. A crate in the emergency section with some pickaxes and manual scanners would be great for ships in a pinch.

**Comment by Sun-Soaked** [comment]
Plus you should maybe document changes like these...

**Comment by tmtmtl30** [comment]
remind me later to update the changelog on this.

**Comment by ZephyrTFA** [comment]
> remind me later to update the changelog on this.

Update the changelog on this

**Pull request merged**